tools/vipsthumbnail.c

```diff
@@ -159,12 +161,12 @@ static GOptionEntry options[] = {
 		G_OPTION_ARG_STRING, &output_format,
 		N_("output to FORMAT"),
 		N_("FORMAT") },
-	{ "export-profile", 'e', 0,
-		G_OPTION_ARG_FILENAME, &export_profile,
+	{ "output-profile", 0, 0,
```

Comment:
Perhaps the short name (i.e. `-e` and `-i`) should be kept on the non-deprecated options? This would ensure that `vipsthumbnail --help` would print these.

master branch:
```console
$ vipsthumbnail --help | grep profile=
  -e, --export-profile=PROFILE     export with PROFILE
  -i, --import-profile=PROFILE     import untagged images with PROFILE
```

This PR:
```console
$ vipsthumbnail --help | grep profile=
  --output-profile=PROFILE      export with PROFILE
  --input-profile=PROFILE       import untagged images with PROFILE
```

Reply:
Maybe, though `-i` and `-e` are not obvious shortcuts.